wp-cli-templates/block-edit-js.mustache

```diff
@@ -21,24 +21,23 @@ export default function {{ className }}Edit( { attributes, className, setAttribu
 	const [ notice, setNotice ] = useState();
 
 	/* Call this function when you want to show an error in the placeholder. */
-	const setErrorNotice = () => setNotice( __( 'Put error message here.', 'jetpack' ) );
+	const setErrorNotice = () => {
+		noticeOperations.removeAllNotices();
+		noticeOperations.createErrorNotice( __( 'Put error message here.', 'jetpack' ) );
+	};
```

Comment:
Part of me wonders if we shouldn't extract this into a shared function available to all blocks? Ideally, that function could also handle all kinds of notices, not just erors. Something like this:

```js
export function displayNotice( message, noticeOperations, type = 'error' ) {
	noticeOperations.removeAllNotices();
	noticeOperations.createNotice( { status: type, content: message } );
}
```

Reply:
That might be worth trying in some blocks, but I'd rather we only move stuff into the scaffolding when we've already had success with it in other blocks.

Reply:
You're right. I suggested we give that a try in https://github.com/Automattic/jetpack/pull/14884#discussion_r388377243

Until then, we should be able to merge your PR and revisit once we have some blocks using it.